user_docs/en/userGuide.md

```diff
@@ -3131,6 +3115,28 @@ This can be very useful when you want to monitor constantly changing content, su
 The refresh takes place every one and a half seconds.
 This option is disabled by default.
 
+#### Set Mirror Dialog {#SetURLDialog}
```

Comment:
I think this should go under "Advanced settings" or "misc settings" since all the other headers at this level are for settings panels

Reply:
I disagree. Putting it in sub-paragraphs of the advanced settings has no sense because this dialog is not accessible from the advanced settings panel. Regarding "Misc", would it be a new paragraph to create?

There are already other level 4 headings which are not settings panels:
* 12.1.3. Select Synthesizer
* 12.1.4. Synth settings ring
12.1.6. Select Braille Display

Though, I acknowledge that Select synthesizer / braille display dialogs are dialogs that can be opened independently as a setting dialog whereas the Set URL dialog cannotn i.e. it is always a sub-dialog of a setting panel.

My proposal is to keep this at level 4, even if it's not a stand-alone settings dialog. But state explicitly in the introduction text of this paragraph that this type of dialog may be opened to set the mirror URL and the add-on store URL.

Reply:
makes a lot of sense, this is quite similar to select synthesizer, settings ring and selecting a braille display. Though perhaps grouping those dialogs with this one separately might be worth doing

Reply:
IMO keeping the paragraphs for synth settings ring, set synth and set braille device next to the related paragraphs (resp. speech settings and braille settings) remains the best organization.

Re the Set URL dialog, I am thinking to something else: make it a sub-paragraph of the "Set mirror option". And for the Set add-on store URL option, just indicate that the "Change" button opens a dialog similar to the one to set the mirror, and link the corresponding sub-paragraph.

Reply:
I had thought about keeping the update mirror URL dialog description and just linking to it from the Add-on Store mirror section (with a note that it's a similar UX), but I was concerned that it might confuse readers. What do you think, @seanbudd, @Qchristensen?

Reply:
I prefer the current solution, or optionally, grouping the popup dialogs separately

Reply:
@seanbudd can you open an issue to group the "child dialogs" separately to get wider feedback?

Reply:
Opened #17238